djangocms_fil_bootstrap/components/workflows.py

```diff
@@ -30,8 +30,12 @@ def workflows(self, roles):
 
     def workflow(self, name, data, roles):
         steps = data.pop("steps", [])
-        workflow = Workflow.objects.create(**data)
-        for step in steps:
-            step["role"] = roles[step["role"]]
-            workflow.steps.create(**step)
-        self.data[name] = workflow
+        workflow_data = {
+            "is_default": data["is_default"],
```

Comment:
So I'm not sure about this because Workflow has more fields than just name and is_default. What's the thinking behind ignoring those fields in the json? Any reason you're not just doing:
```
name = data.pop("name")
workflow, created = Workflow.objects.get_or_create(name=name, defaults=data)
```

Reply:
ok changed